HomeTask2.7__webpack-demo/src/services/getTopHeadlines.js

```diff
@@ -0,0 +1,21 @@
+import generateParamUrl from "../utils/generateParamUrl";
+import getDateTime from "../utils/getDateTime";
+
+const getTopHeadlines = async (api) => {
+  const requestParams = {
+    country: `in`,
+  };
+  const paramUrl = `top-headlines?${generateParamUrl(requestParams)}`;
+  return api(paramUrl).then(
+    (response) => {
+      console.log(response);
```

Comment:
Remove console logs

Reply:
That was one of the requirements to remove console.log in production code. So I deliberately added it